controller/kubernetes_pod_controller.go

```diff
@@ -199,6 +200,28 @@ func (kc *KubernetesPodController) handlePodDeletionIfNodeDown(pod *v1.Pod, node
 		return nil
 	}
```

Comment:
Should we move the condition check `pod.DeletionTimestamp.After(time.Now())` in line 198/199 to the next of the VA deletion?

Reply:
Could you explain a bit more about why?

Reply:
When this check is passed, it means Longhorn decides to delete or is already deleting the pod forcefully. But this PR aims to do the VA deletion before the pod force deletion.
The best timing for the VA deletion is after the pod deletion timestamp is set but before the force deletion starts.

Reply:
@PhanLe1010 any concerns?

Reply:
Make sense that VA deletion is after the pod deletion timestamp is set but before the force deletion starts! Thank you for the explanation! I pushed the change